presence/package.json

```diff
@@ -28,14 +28,13 @@
   },
   "dependencies": {
     "minimist": ">=0.2.1",
-    "cloudstate": "0.0.3"
+    "cloudstate": "0.0.3",
+    "@grpc/proto-loader": "^0.1.0",
+    "google-protobuf": "^3.0.0",
+    "grpc": "^1.20.2"
```

Comment:
I think we decided to remove these packages from explicit dependencies since the `cloudsate` package depends on them.

Reply:
Grrr, good catch